org.civicrm.reqangular/src/common/services/file-upload.js

```diff
@@ -0,0 +1,81 @@
+define([
+  'common/modules/services',
+], function (module) {
+  'use strict';
+
+  module.factory('FileUploadService', ['$q', 'FileUploader',
```

Comment:
Isn't `FileUploader` an external dependencies registered on a different module? Doesn't it mean then that `common.service` should have a dependency on that module?